src/BPEssentials/Commands/Player/ToggleMute.cs

```diff
@@ -0,0 +1,25 @@
+﻿using BPEssentials.Abstractions;
+using BPEssentials.ExtensionMethods;
+using BrokeProtocol.Entities;
+using BrokeProtocol.Utility;
+
+namespace BPEssentials.Commands
+{
+    class ToggleMute : Command
```

Comment:
Keep the accessibility consistent. Commands should always be public.
```suggestion
    public class ToggleMute : Command
```